server/game/cards/03_TWI/units/WolfPackEscort.ts

```diff
@@ -0,0 +1,27 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { CardType, RelativePlayer, Trait } from '../../../core/Constants';
+
+export default class WolfPackEscort extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '1882027961',
+            internalName: 'wolf-pack-escort'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'You may return a friendly non-leader, non-Vehicle unit to its owner\'s hand.',
```

Comment:
We omit the "you may" part for our titles since there will be a "pass" button that appears, indicating that it is optional. So this can just be `Return a friendly non-leader, non-Vehicle unit to its owner\'s hand.`